addons/animated-thumb/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Thumbnails setter",
-  "description": "Adds a button to the editor to set the thumbnail of your project to any image (including gifs).",
+  "description": "Adds a button to the editor menu bar to set the thumbnail of your project to any image (including gifs).",
```

Comment:
This might also change soon because of #2334

Reply:
Same as above - until it gets moved somewhere else, the exact location of the button should be mentioned.